### PR TITLE
Added support for omitting country prefix from short codes

### PR DESCRIPTION
**Pull request by duane**
@scootklein @detkin @mrhamric @tylerdavis @sjohnson540 

Previously, `Hermes::Phone#full_number` prints the country prefix (eg `+1`) along with long code (e.g. `555-555-5555`). This omits the prefix for short codes (e.g. `12345`).
- [x] Code change
- [x] Unit test

---

[Asana Task](https://app.asana.com/0/41333918339141/130820207131467)

### CODE REVIEW

File: lib/support/phone.rb [hunk 1]
```diff
@@ -3,6 +3,12 @@ class Phone
 
     attr_accessor :country, :number
 
+    CODE_LENGTH_RANGE_BY_COUNTRY_CODE = {
```
Comment (scootklein) [inline lib/support/phone.rb:6]: not sure we need this forward-looking planning structure if all of them are the same

`SHORT_CODE_LENGTH_RANGE = 5..6`

File: lib/support/phone.rb [hunk 2]
```diff
@@ -222,7 +228,17 @@ def country_name
...
-      self.class.prefix_for_country(self.country) + self.number
+      if self.short_code?
+        self.number
+      else
+        self.class.prefix_for_country(self.country) + self.number
+      end
+    end
+
+    def short_code?
+      range = CODE_LENGTH_RANGE_BY_COUNTRY_CODE[self.country]
```
Comment (scootklein) [inline lib/support/phone.rb]: can consolidate this with `try`

``` ruby
!!CODE_LENGTH_RANGE_BY_COUNTRY_CODE[self.country].try(:include?, self.number.length)
```

File: test/lib/support/extractors_test.rb [hunk 3]
```diff
@@ -77,6 +77,12 @@ class SomeBasicWrapper
       assert_equal @wrapper.extract_from(@texter_message, format: :address), from
     end
 
+    it "handles short codes" do
+      from = Hermes::Phone.new("us", "12345")
+      from.full_number.must_equal "12345"
+      from.short_code?.must_equal true
```
Comment (scootklein) [inline test/lib/support/extractors_test.rb:83]: should test the negative case here as well

``` ruby
from = Hermes::Phone.new("af", "12345")
from.full_number.must_equal "+9312345"
from.short_code?.must_equal false
```

File: lib/support/phone.rb (hunk 1 above)
Comment (scootklein) [inline lib/support/phone.rb:6]: ah, i see on the countries that are supported, nvm on this

Comment (scootklein) [inline lib/support/phone.rb:6]: one final note, we use a class variable below with the `@@` syntax, either do the same here or change the `@@countries` variable to be a constant, just want to maintain style

Comment (mrhamric) [inline lib/support/phone.rb:6]: let's try to avoid global vars if possible? this definitely seems like a constant to me

**Pull request merged**